mon-pix/app/components/authentication/oidc-provider-selector.scss

```
@@ -0,0 +1,46 @@
+.oidc-provider-selector {
+  .pix-select-button {
+    padding: var(--pix-spacing-3x);
+    border-color: var(--pix-neutral-500);
+
+    .pix-select-button__text {
+      color: var(--pix-neutral-500);
+    }
+  }
+
+  .pix-select__search {
+    margin: var(--pix-spacing-2x) var(--pix-spacing-3x);
+    background: var(--pix-neutral-0);
+    border-bottom-width: 0;
+
+    input {
+      background: var(--pix-neutral-0);
+    }
+  }
+
+  .pix-select__dropdown {
+    max-height: 488px;
+  }
+
+  .pix-select-options-category__option {
+    padding: var(--pix-spacing-2x) var(--pix-spacing-3x);
+
+    &:first-child {
+      display: none;
+    }
+
+    &:hover, &:focus {
+      background: var(--pix-primary-100);
+    }
+
+    &:nth-child(even) {
```

```suggestion
    &:nth-child(odd) {
```

A cause du 1er élément (option par défaut) qu'on cache